commcare_connect/opportunity/tests/test_models.py

```diff
@@ -27,29 +27,37 @@ def test_learn_progress(opportunity: Opportunity):
 
 
 @pytest.mark.django_db
+@pytest.mark.parametrize("opportunity", [{}, {"opp_options": {"managed": True}}], indirect=True)
 def test_opportunity_stats(opportunity: Opportunity, user: User):
-    payment_unit_sub = PaymentUnitFactory(
+    payment_unit_sub = PaymentUnitFactory.create(
         opportunity=opportunity, max_total=100, max_daily=10, amount=5, parent_payment_unit=None
     )
-    payment_unit1 = PaymentUnitFactory(
+    payment_unit1 = PaymentUnitFactory.create(
         opportunity=opportunity,
         max_total=100,
         max_daily=10,
         amount=3,
         parent_payment_unit=payment_unit_sub,
     )
-    payment_unit2 = PaymentUnitFactory(
+    payment_unit2 = PaymentUnitFactory.create(
         opportunity=opportunity, max_total=100, max_daily=10, amount=5, parent_payment_unit=None
     )
     assert set(list(opportunity.paymentunit_set.values_list("id", flat=True))) == {
         payment_unit1.id,
         payment_unit2.id,
         payment_unit_sub.id,
     }
+    payment_units = [payment_unit_sub, payment_unit1, payment_unit2]
+    budget_per_user = sum(pu.max_total * pu.amount for pu in payment_units)
+    org_pay = 0
+    if opportunity.managed:
```

Comment:
Shouldn't we know if the opportunity is managed or not in the test?

Reply:
Yes, we have that information available, this line on the top of this test function creates 1 case with a normal opportunity and another with a managed opportunity.
`@pytest.mark.parametrize("opportunity", [{}, {"opp_options": {"managed": True}}], indirect=True)`